src/Uno.UI/UI/Xaml/Controls/NavigationView/NavigationView.Properties.cs

```diff
@@ -530,7 +530,7 @@ public NavigationViewTemplateSettings TemplateSettings
 			nameof(PaneTitle),
 			typeof(string),
 			typeof(NavigationView),
-			new FrameworkPropertyMetadata(default(string),
```

Comment:
In UWP this is not a DP. I'll keep it as string.Empty since this is the default of the regular property.